From 308705710d40ced3ea2a913d24ab99cc61bef01f Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Sun, 5 Apr 2015 18:11:25 +0200
Subject: [PATCH] pkg-config: needs pthread as private library

libtirpc needs pthreads. This is automatically pulled in when linking
with the shared library, thanks to the DT_NEEDED ELF tags.

But for a static library, there is no such mechanism to identify
required libraries.

Thus, add -pthread to the private libraries in the pkg-config file.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
---
Patch sent upstream:
    http://sourceforge.net/p/libtirpc/mailman/message/33072076/

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 libtirpc.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libtirpc.pc.in b/libtirpc.pc.in
index ec4cf75..1dc85ea 100644
--- a/libtirpc.pc.in
+++ b/libtirpc.pc.in
@@ -8,4 +8,5 @@ Description: Transport Independent RPC Library
 Requires:
 Version: @PACKAGE_VERSION@
 Libs: -L@libdir@ -ltirpc
+Libs.private: -pthread
 Cflags: -I@includedir@/tirpc
-- 
2.1.0

